extensions/adobe/experience/campaign/journeyaifatigue.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/journeyaifatigue",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey AI Fatigue",
```

Comment:
while this is part of an Adobe extension - is the naming here too product and use specific?

Reply:
I will change the title to be less specific about Journey AI. Should I have to change the id ( `journeyaifatigue` ) as well?